spec/requests/api/v1/course_spec.rb

```diff
@@ -186,11 +226,13 @@ def get_course_codes_from_body(body)
 
 
         expect(response.headers).to have_key "Link"
-        url = url_for(params: {
-                        changed_since: timestamp_of_last_course.utc.strftime('%FT%T.%6NZ'),
-                        per_page: 100,
-                        recruitment_year: 2019
-                      })
+        url = url_for(
+          recruitment_year: 2019,
+          params: {
+            changed_since: timestamp_of_last_course.utc.strftime('%FT%T.%6NZ'),
+            per_page: 100
+          }
```

Comment:
Oh, one more nice-to-have, how about making this the full explicit URL? So it's clear exactly what this should look like.

Reply:
Have experimented. Can't get it work due to string matching not working. Asked Sau. Both couldn't figure it out. Don't want to waste more time on a nice to have.